### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by ianuragsharma**
image

### CODE REVIEW

**Comment by arpit01923** [comment]
LGTM

File: styles/card.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <link rel="stylesheet" href="card.css">
+    <script src="https://kit.fontawesome.com/f45a42e7ea.js" crossorigin="anonymous"></script>
+</head>
+
+
+<body>
+    <!-- vertical card  -->
+    <div class="vertical-card">
+
+        <img src="https://rb.gy/pwrsjk" alt="">
```
Comment (Aditi0505) [inline styles/card.html]: Please provide alt value for accessibility.

**Review by Aditi0505 — APPROVED** [review]
Have suggested to provide values in alt attribute. This is missing in many places. It would be a good thing to add for accessibility.

**Review by smilegupta — APPROVED** [review]
LGTM

**Pull request merged**